src/components/v2/V2Create/tabs/TokenTab/TokenTabContent.tsx

```diff
@@ -215,7 +215,7 @@ export default function TokenTabContent({
               tokenForm.setFieldsValue({ redemptionRate: val?.toString() })
             }
             style={{ ...shadowCard(theme), padding: 25 }}
-            label={t`Redemption rate`}
+            label={t`Bonding curve`}
```

Comment:
**Issue:** I might not be up to date, but pretty sure a week or so ago we came to a consensus with Jango that we wanted to overhaul "Bonding curve"s to "Redemption rate"s with V2 😀 may have missed something though

Reply:
ahh oh got it, i missed that conversation so prob your understanding is the latest, but i thought redemption rate means the redeem rate? will people be confused between redeem rate vs redemption rate?

Reply:
100% agree, but in accordance with what we've agreed to so far, I've reverted it back for now. Definitely more discussion to be had around it though